rts/at/util/src/main/java/org/jboss/jbossts/star/util/TxSupport.java

```diff
@@ -547,6 +550,7 @@ private void addLocationHeader(HttpURLConnection connection, Map<String, String>
             if (connection.getResponseCode() == HttpURLConnection.HTTP_CREATED)
                 links.put("location", connection.getHeaderField("location"));
         } catch (IOException e) {
+        	RESTATLogger.at18NLogger.warn_rest_at_txSupportAddLocationHeader(e.getMessage(),e);
```

Comment:
similar to what I said for `warn_rest_at_txSupportHttpRequest` - this seems to be "expected error" state. The logging of the exception should go with `debug` or `trace` but not with warning.

Reply:
The method is adding the Location Header to the map that the client passed in. The client may well be relying on the Location header to be copied into the map. So I think the original code should have logged a warning here and Sweta's addition of the warning is necessary.

Reply:
Done